src/compiler/diagnosticMessages.json

```diff
@@ -2056,6 +2056,10 @@
         "category": "Error",
         "code": 2574
     },
+    "No overload expects {0} arguments, but overloads do exist that expect either {1} or at least {2} arguments.": {

```

Comment:
I added your simplified error message, however I feel like it might hide information from the user in cases where discontinuous overload sets with argument counts less than the current argument count exist.